includes/course-theme/class-sensei-course-theme.php

```diff
@@ -13,14 +13,20 @@
 use \Sensei\Blocks\Course_Theme;
 
 /**
- * Sensei_Course_Theme class.
+ * Load the 'Sensei Course Theme' theme for the /learn subsite.
  *
- * @since 3.13.4
+ * @since 4.0.0
  */
 class Sensei_Course_Theme {
-	const THEME_POST_META_NAME = '_course_theme';
-	const WORDPRESS_THEME      = 'wordpress-theme';
-	const SENSEI_THEME         = 'sensei-theme';
+	/**
+	 * URL prefix for loading the course theme.
+	 */
+	const QUERY_VAR = 'learn';
```

Comment:
Didn't want to complicate it for now, since the rewrite rules also need to be flushed anytime this changes. So I think for that reason a filter wouldn't work — I was thinking to store it in an option when we allow customization.

Reply:
Good point! Makes sense!